assessment.md

```diff
@@ -202,39 +202,78 @@ TODO: add description for each assessment level for this skill.
 
 * Attendance
 
-TODO: add description for each assessment level for this skill.
+  * None: Student attended 0% of meetings.
+  * Inadequate: Student attended 30% of meetings.
+  * Adequate: Student attended 50% of meetings.
+  * Good: Student attended 70% of meetings.
+  * Excellent: Student attended 100% of meetings.
 
 * Deadlines
 
-TODO: add description for each assessment level for this skill.
+  * None: This group member has no disregard for the deadline and ignored it.
+  * Inadequate: This group member acknowledged the deadline but did not manage their time to meet it.
+  * Adequate: This group member acknowledged the deadline and managed their time to meet the deadline, but the work was sloppy and  unorganized.
+  * Good: This group member kept a reminder of the deadline and managed their time to meet the deadline, and the work was orderly.
+  * Excellent: This group member kept constant reminders of the deadline and designated time every day to meet the deadline, asked for  any extensions if needed, and their work was very clean and organized.
 
 * Communication
 
-TODO: add description for each assessment level for this skill.
+  * None: Student does not interact with others, and neglects group members.
+  * Inadequate: Student responds to group members with great delay, and lacks contributions.
+  * Adequate: Student interacts with the group moderately and is in proper communication.
+  * Good: Student helps other group members and interacts frequently with the group.
+  * Excellent: Student leads the group discussions and helps other students.
 
 * Participation
 
-TODO: add description for each assessment level for this skill.
+  * None: Student does not do any work or show up to anything zero communication
+  * Inadequate: Student does no work and barely shows up with zero communication
+  * Adequate: Student does some work and shows up and communicates
+  * Good: Student does part and shows up and communicates effectively
+  * Excellent: Student does part and works effectively and efficiently with group members. Communications effectively
+
 
 * Conduct
 
-TODO: add description for each assessment level for this skill.
+  * None: Student physically attacks other students and yells at them, insults and acts rudely towards other students. Sabotages the project
+    if student in group does not get their way
+  * Inadequate: Student tries to pick a fight with everyone, acts rudely and is unaccepting of students. Refuses to work with others.
+  * Adequate: Student follows rules and works with everyone and is respectful.
+  * Good: Student is kind and respectful. Openminded and willing to learn. Works with everyone.
+  * Excellent: Student kind and respectful. Openminded and eager to learn. Works with everyone and is helpful
```

Comment:
I think it would also be good to add whether the student follows the code of conduct or not